estate/models/estate_property.py

```diff
@@ -78,3 +78,75 @@ def _onchange_garden(self):
             self.garden_area = 0
             self.garden_orientation = False
 
+
+    def action_sold(self):
```

Comment:
This is what I meant by rebasing, it looks like you are adding these methods in this commit

but they already exists [here](https://github.com/Cindypsd/Odoo_Training/blob/main/estate/models/estate_property.py)

If the changes are only the position of the method we an resolve this comment, just be aware of this please